src/core/infra/repositories/apiConfigBuilders.ts

```diff
@@ -13,11 +13,14 @@ export const buildRequestConfig = (
     headers: {
       'Content-Type': contentType
     },
+    withCredentials: false,
```

Comment:
In` AxiosRequestConfig`, the default value of `withCredentials` is `false`, so would it be cleaner to not set this here, and only set the value to true for the SESSION_COOKIE use case?  It works fine as is, it was just a little confusing to me at first.